[2024-07-23 11:11:25,814] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.thrid_task 2024-07-19T00:00:00+00:00 [queued]>
[2024-07-23 11:11:25,997] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.thrid_task 2024-07-19T00:00:00+00:00 [queued]>
[2024-07-23 11:11:26,002] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-07-23 11:11:26,008] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2024-07-23 11:11:26,010] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-07-23 11:11:26,138] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): thrid_task> on 2024-07-19T00:00:00+00:00
[2024-07-23 11:11:26,237] {standard_task_runner.py:52} INFO - Started process 342 to run task
[2024-07-23 11:11:26,251] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v5', 'thrid_task', '2024-07-19T00:00:00+00:00', '--job-id', '12492', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpw02azjns', '--error-file', '/tmp/tmp6fjih65a']
[2024-07-23 11:11:26,263] {standard_task_runner.py:77} INFO - Job 12492: Subtask thrid_task
[2024-07-23 11:11:26,899] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v5.thrid_task 2024-07-19T00:00:00+00:00 [running]> on host 799b1e36e3f1
[2024-07-23 11:11:27,602] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=thrid_task
AIRFLOW_CTX_EXECUTION_DATE=2024-07-19T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-07-19T00:00:00+00:00
[2024-07-23 11:11:27,604] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2024-07-23 11:11:27,645] {bash.py:158} INFO - Running command: echo hey, I am task3 and will be running after task1 at the same time as task2!
[2024-07-23 11:11:27,690] {bash.py:169} INFO - Output:
[2024-07-23 11:11:27,696] {bash.py:173} INFO - hey, I am task3 and will be running after task1 at the same time as task2!
[2024-07-23 11:11:27,705] {bash.py:177} INFO - Command exited with return code 0
[2024-07-23 11:11:27,948] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=thrid_task, execution_date=20240719T000000, start_date=20240723T111125, end_date=20240723T111127
[2024-07-23 11:11:28,354] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-23 11:11:28,548] {local_task_job.py:146} INFO - Task exited with return code 0
